api/src/pcapi/alembic/versions/20240523T073045_73792a10a8f6_add_collective_stock_start_end_datetime.py

```diff
@@ -0,0 +1,23 @@
+"""add_collective_stock_start_end_datetime
```

Comment:
En théorie, il faut un titre plus joli, du genre de
`Add startDatetime and endDatetime columns to collectiveStock`